lib/nifty_services/base_service.rb

```diff
@@ -188,7 +188,9 @@ def error!(status, message_key, options = {})
     end
 
     def valid_object?(record, expected_class)
-      record.present? && record.is_a?(expected_class)
+      blank = record.respond_to?(:empty?) ? !!record.empty? : !record
```

Comment:
No problem, I say, if validate the object before checking if is a valid class, so lets suppose we what to check this:

```ruby
def valid_record?(record, expected_class)
  # or !record.nil? && record.is_a?(expected_class)
  record.is_a?(expected_class)
end

valid_record?(User.new, User) # true
valid_record?(false, User) # false
valid_record?(nil, User) # false
valid_record?({}, User) # false
valid_record?([], User) # false
```

We don't need to validate if `record` object is `valid` before checking if is from the kind of class we what to check, cuz if it's `invalid`(nil, false, []), `is_a?` will work as expected, makes sense?